modules/react-map-gl-draw/README.md

```diff
@@ -1,4 +1,4 @@
-[![docs](https://i.imgur.com/BTVrsR4.jpg)](https://neb.gl)
+[![docs](https://i.imgur.com/bRDL1oh.gif)](https://nebula.gl)
```

Comment:
Again I would recommend keeping the README in each module minimal, and move the text into the 	`docs/modules/...`

Otherwise the risk that this doc (which will be published on npm) diverges from the corresponding doc in the website is very big.

And I would expect that anything that is documented in the npm doc is also in the main docs.